previews/primer/alpha/radio_button_preview.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    # @label RadioButton
+    class RadioButtonPreview < ViewComponent::Preview
+      # @label Playground
+      #
+      # @param name text
+      # @param id text
+      # @param value text
+      # @param label text
+      # @param caption text
+      # @param visually_hide_label toggle
+      # @param full_width toggle
+      # @param disabled toggle
+      def playground(
+        name: "my-radio-button",
+        id: nil,
+        value: "bsg",
+        label: "Battlestar Galactica",
+        caption: "A pretty good show",
+        visually_hide_label: false,
+        full_width: false,
```

Comment:
Also remove full width from radio (both checkbox and radio dont need this param)